Install-Anaconda.md

```diff
@@ -0,0 +1,75 @@
+## Installation Guide for Priism in Anaconda
+
+### Notes
```

Comment:
I guess you implicitly assume that the reader of this guide is using either [miniforge](https://github.com/conda-forge/miniforge) or miniconda with customized channel that directs to conda-forge. Could you clarify this in the Notes section? 

As you know, Anaconda changed its policy recently so that even non-profit organization requires paid license if its employees/contractors are more than 200. I want to avoid any confusion and/or suspected use of Anaconda.